### DISCUSSION TITLE
yaml-pipelines-part2

### DISCUSSION THREAD

**Discussion by giscus** [WorkSafeBC-Common-Engineering/Common.Engineering.Discussion#14] [Technical Blog]
# yaml-pipelines-part2

https://wsbctechnicalblog.github.io/yaml-pipelines-part2.html

**Comment by JimSander-uniphore** [comment]
Good stuff! Minor: update `master` to `main` in `It is evident that: - It is triggered on changes on the master branch`

**Reply by wpschaub** [reply]
Thanks for spotting the gremlin Jim. I will queue an update today.

**Comment by Himank-Kadian** [comment]
Enjoying learning this so far! Thanks for this.

Real quick, I believe the last line under `Parameters` should be: The <ins>parameter</ins> is processed at queue time.

**Reply by wpschaub** [reply]
Thanks for spotting another gremlin! I will fix this with my next blog post later today.
